app/app.go

```diff
@@ -103,17 +103,17 @@ func (a *DeviceConnectApp) GetDevice(
 	return device, nil
 }
 
-// DeleteDevice provisions a new tenant
+// DeleteDevice decomissions a new tenant
```

Comment:
technically it deletes a device, not a new tenant.

Reply:
Right: https://github.com/mendersoftware/deviceconnect/pull/15/commits/73301349b32549676e34eec8cb4cb2490a0a4c3c